content/links.md

```diff
@@ -227,13 +227,76 @@ instance_groups:
       conn: {as: db_conn}
 ```
 
+### Explicit Linking {: #explicit }
+Explicit linking is defined as specifying your consumer in your deployment manifest. The consumer will be matched on both name and type.
+
+##### Providers
+Explicitly defined providers which can be both explicitly and implicitly consumed, can have the following optional properties:
+
+* **as** [String]: Overrides the name of the provider defined in the release spec. Example:
+```yaml
+instance_groups:
+- name: my_instance_group
+  jobs:
+  - name: my_provider_job
+    provides:
+      bar: {as: new_name}
+```
+* **shared** [Boolean]: *Default is false* Sets whether this provider is consumable from another deployment. See [cross deployment links](#cross-deployment).
+
+Providers that are specified as `nil` will cause your *explicit* consumer to fail resolution.
```

Comment:
> Providers that are specified as `nil` will cause your *explicit* consumer to fail resolution.

Probably this can be reworded to mention that specifying `nil` for the provider in the manifest will block it from ever being consumed (implicitly or explicitly). There is no benefit to mention what will happen to the consumer, it is technically up to the the operator to figure out where the consumer needs to get the link from.

Reply:
Will update.